applications/luci-app-cifsd/Makefile

```diff
@@ -3,7 +3,7 @@
 include $(TOPDIR)/rules.mk
 
 LUCI_TITLE:=Network Shares - CIFSD CIFS/SMB kernel fileserver
```

Comment:
I currently have this installed and it shows up as Network Shares. I imagine if I had luci-app-samba installed it would be the same. Is this a bug?

Reply:
Yes you should have two "Network Shares" entries than, i was considering adding something like:
```
Network Shares (Cifsd)
Network Shares (Samba4)
```
Yet i think this is a special case if someone has both servers installed? We can add the app name, but i'm not sure if this is necessary?

Reply:
Jow noted we can ignore this atm and i added naming/version infos to the actual network pages so its easy to spot for what server you are making changes.